docs/docs/05-reusable-components.md

```diff
@@ -296,6 +296,21 @@ ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);
 This simplified component API is intended for components that are pure functions of their props. These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods. They are pure functional transforms of their input, with zero boilerplate.
 However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class.
 
+```javascript
+const HelloMessage = (props) => <div style={props.style} className={props.className}>Hello, {props.name}</div>;
```

Comment:
I would use deconstruction. I would also not be afraid to break it up into multiple lines for better readability. A "single statement" fat arrow function does not mean single line.

Instead of:
```js
const HelloMessage = (props) => <div style={props.style} className={props.className}>Hello, {props.name}</div>;
```

Your example is better represented (IMO) as follows:
```js
const HelloMessage = ({ style, className, name }) => (
    <div style={style} className={className}>
        Hello, {name}
    </div>
);
```
But then again, I could be wrong. 😄